jira/manifest.yml

```diff
@@ -0,0 +1,23 @@
+modules:
+  trigger:
+    - key: view-activity
+      function: view
+      events:
+        - avi:jira:viewed:issue
+  function:
+    - key: view
+      handler: index.eventHandler
+app:
```

Comment:
`app` will get updated with the new ID on running [forge create](https://developer.atlassian.com/platform/forge/automate-jira-using-triggers/#step-1--create-your-app)